src/Microsoft.ML.EntryPoints/FeatureCombiner.cs

```diff
@@ -266,10 +267,10 @@ public static CommonOutputs.TransformOutput ConvertPredictedLabel(IHostEnvironme
             host.CheckValue(input, nameof(input));
             EntryPointUtils.CheckInputArgs(host, input);
 
-            int predictedLabelCol;
-            if (!input.Data.Schema.TryGetColumnIndex(input.PredictedLabelColumn, out predictedLabelCol))
+            var predictedLabelCol = input.Data.Schema.GetColumnOrNull(input.PredictedLabelColumn);
+            if (!predictedLabelCol.HasValue)
                 throw host.Except($"Column '{input.PredictedLabelColumn}' not found.");
```

Comment:
>Except [](start = 27, length = 6)

I'm not sure if you want to do this at the same time, but in the general movement to start using `ExceptSchemaMismatch` to ensure greater uniformity between error messages, you might consider doing that transition at the same time. But, not sure if you want to.